templates/base.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load donation_tags %}
```

Comment:
Did you intentionally add this?

Reply:
Yes, it's where the `settings_value` template tag comes from. It's not a native Django tag, but rather a hack added by GDQ to this project.